Fix formatting for `if` clauses in `match-case` blocks (#4269)
psf/black: docs/the_black_code_style/future_style.md

  quotes of a docstring
- `remove_redundant_guard_parens`: Removes redundant parentheses in `if` guards for
  `case` blocks.

> - `parens_for_long_if_clauses_in_case_block`: Fixed a bug where `if` clauses in
>   `match-case` blocks are not wrapped with parenthesis when the line is too long

Reviewer: ```suggestion
- `parens_for_long_if_clauses_in_case_block`: Adds parentheses to `if` clauses in
  `case` blocks when the the line is too long
```
